packages/tests-e2e/tests/appRouter/revalidateTag.test.ts

```diff
@@ -79,6 +79,8 @@ test("Revalidate path", async ({ page, request }) => {
   elLayout = page.getByText("Date:");
   const initialDate = await elLayout.textContent();
 
+  await page.waitForTimeout(2000);
```

Comment:
nit: add a comment about why this is needed/what it solves?

Reply:
```suggestion
  // Wait so that enough time passes for the data on the page to update when revalidating
  await page.waitForTimeout(2000);
```